tests/__fixtures__/Rules/lowercase_description/fixture.js

```diff
@@ -0,0 +1,169 @@
+const { format } = require("util");
+const generator = require("../../../helpers/problemGenerator");
+const LowercaseDescription = require("../../../../lib/rules/lowercase_description");
+
+function generateProblem(location, keyword) {
+    return generator(
+        LowercaseDescription,
+        location,
+        format(LowercaseDescription.meta.message, keyword),
+        {
+            applyFix: jest.fn(),
+        }
+    );
+}
+
+function getValidTestData() {
+    return [
+        [
+            "Feature - Rule - Background - Scenario",
+            `Feature: a feature file
+  Rule: a rule
+    Background: a background
+    Scenario: a scenario`,
```

Comment:
we can put them all in one and only have two data set for valid cases
```suggestion
            `Feature: a feature file
  Rule: a rule
    Background: a background
    Scenario: a scenario`
    Scenario Outline: a scenario,
```

Reply:
okay